src/pages/definition/builder/JsonPanel.tsx

```diff
@@ -130,7 +130,8 @@ export default function JsonPanel() {
         setDecorations([]);
       }
     }
```

Comment:
Curious, what dependency is being complained about that caused you to add the eslint disable?

Reply:
`decorations` is being set endlessly in a loop. This is an edge case in that the monaco editor is not really a controlled component that depends on the state of `selectedTask`, but we only care about the transitions. It could probably be rewritten in a better form without the eslint override but I think the logic here is sound for the purpose of suppressing an active breakage in prod.